concepts/type-theory-glossary.md

```diff
@@ -216,8 +216,13 @@ This "open membership" is commonly known as **interfaces**.
 ### interfaces
 
 Interfaces have the concept of "open membership", and are also typically
-combined with some definition of "features" that are required if something
-will be considered to implement the interface.
+combined with some definition of "features", also called as traits or behavior
+in some languages, that are required if something will be considered to
+implement the interface.
+
+Like all values, an interface also have well-defined memory footprint in
```

Comment:
"an interface will also have ..."

Reply:
This paragraph is a bit... far altogether.  I'd think I'd rather take it back out.

The rest of this document stays pretty far away from talking about memory layouts (certainly at least until it gets to the "in language X..." parts, much later), so it's odd to bring up memory footprints here.

I also just don't think I'd say interfaces have a well-defined memory footprint in a program.  That's a pretty expansive claim to try to make for all programing languages.